src/server/routes/users.js

```diff
@@ -105,22 +105,25 @@ router.post('/create', adminAuthMiddleware('create a user.'), async (req, res) =
 			role: {
 				type: 'string',
 				enum: Object.values(User.role)
+			},
```

Comment:
Should note be a required param?

Reply:
I didn't make it required as I figured you could have no note, but I'll add it as required since it seems we are going to be using en ampty string as the default now.